src/query/storages/parquet/src/parquet_table/non_blocking.rs

```diff
@@ -27,28 +28,37 @@ use opendal::Operator;
 
 use super::table::create_parquet_table_info;
 use crate::ParquetTable;
-use crate::ReadOptions;
 
 impl ParquetTable {
     pub async fn create(
         table_id: u64,
-        table_args: TableArgs,
         operator: Operator,
         maybe_glob_locations: Vec<String>,
-        read_options: ReadOptions,
+        read_options: ParquetReadOptions,
+        stage_info: Option<UserStageInfo>,
     ) -> Result<Self> {
+        if operator.metadata().can_blocking() {
```

Comment:
I think this check should be outside `create`.

Reply:
ok, I intend to avoid same check when `from_info`, 
but seem `from_info` does not need to check.

but I prefer the check at parquet_table side,

what about create, nonblocking_create, blocking_create